app/src/main/java/com/usepace/android/messagingcenter/screens/sendbird/SendBirdChatActivity.java

```diff
@@ -192,12 +192,19 @@ public void finish() {
     protected void onDestroy() {
         setResult(MessageCenter.OPEN_CHAT_VIEW_RESPONSE_CODE);
         super.onDestroy();
-        SendBird.disconnect(new SendBird.DisconnectHandler() {
```

Comment:
there is a need for it to enable notifications when the user close the chat view, Make sure you test this edge case

Reply:
I have updated and pushed the code

Reply:
done